src/Spinners/LoadSpinnerText.tsx

```diff
@@ -0,0 +1,41 @@
+import React, {
+  FunctionComponent, ReactElement, useContext,
+} from 'react';
+import styled, { ThemeContext } from 'styled-components';
+import { BaseTheme } from '../Theme';
+
+export interface LoadSpinnerTextProps {
+  /** Specifies the text under the spinner */
+  children: string;
+  /** Specifies the color of the text under the spinner */
+  color?: string;
+}
+
+const StyledLoadSpinnerText = styled.div`
+  background: transparent;
+  color: ${({ color }): string => (color)};
```

Comment:
Rather than allowing _any_ color value here, I'd prefer to provide a limited set of options with an `enum`, and to have those options correspond to the `light` and `dark` values in our theme. To set that up, you could do something like:

```tsx
export enum SPINNER_TEXT {
  LIGHT = 'light',
  DARK = 'dark',
}

export interface LoadSpinnerTextProps {
  /** Specifies the text under the spinner */
  children: string;
  /** Specifies the color of the text under the spinner */
  color?: SPINNER_TEXT;
}

const StyledLoadSpinnerText = styled.div<LoadSpinnerTextProps>`
  color: ${({ color, theme }): string => theme.color.text[color]};
  ...
`

StyledLoadSpinnerText.defaultProps = {
  color: SPINNER_TEXT.DARK;
}
```

The idea here being that we want to keep text colors consistent across the library (by using our only our defined colors), and we want to limit how much "damage" (in terms of inaccessible design) we let people do by only letting them use white or black text.

Reply:
make sense thought light color will be invisible on a white background